runtime/j9vm/jvm.c

```diff
@@ -902,36 +931,30 @@ BOOL APIENTRY DllMain(HANDLE hModule, DWORD  ul_reason_for_call, LPVOID lpReserv
 	return rcDllMain;
 }
 
-
 static BOOLEAN
 preloadLibraries(void)
 {
-	char* tempchar = 0;
+	char *tempchar = NULL;
 	char jvmDLLName[J9_MAX_PATH];
-	J9StringBuffer * jvmDLLNameBuffer = NULL;
-	wchar_t unicodeDLLName[J9_MAX_PATH + 1] = {0}; /*extra character is added to check if path is truncated by GetModuleFileNameW()*/
+	J9StringBuffer *jvmDLLNameBuffer = NULL;
+	wchar_t unicodeDLLName[J9_MAX_PATH + 1] = {0}; /* extra character is added to check if path is truncated by GetModuleFileNameW() */
 	DWORD unicodeDLLNameLength = 0;
 	UINT prevMode;
 	HINSTANCE vmDLL = 0;
 	HINSTANCE threadDLL = 0;
 	HINSTANCE portDLL = 0;
-	char* vmDllName = J9_VM_DLL_NAME;
-	char* vmName = NULL;
-	static beenRun=FALSE;
-	if(beenRun) {
-		return TRUE;
-	}
-	beenRun = TRUE;
```

Comment:
This needs to stay so this function is idempotent, but the type should be explicit:
```c
	static BOOLEAN beenRun = FALSE;
	if (beenRun) {
		return TRUE;
	}
	beenRun = TRUE;
```